portfolio/src/main/java/com/google/sps/servlets/PieChartServlet.java

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet stores and retrieves data about favorite milk tea flavors. */
+@WebServlet("/pie-chart")
+public class PieChartServlet extends HttpServlet {
```

Comment:
nit: consider the naming - does this servlet actually have anything to do with pie charts? i could take the data from this servlet and display a bar chart, or anything else really. this servlet is responsible for returning info on boba flavors, and the naming should reflect that.

Reply:
I changed the name of the servlet to TeaVoteServlet. I also changed some other variables for the same reason that the servlet does not have anything to do with the creation of the pie chart.